CHANGES.md

```diff
@@ -39,6 +39,7 @@ Grammar improvements:
 - enh(dart) Fix empty block-comments from breaking highlighting (#2898) [Jan Pilzer][]
 - enh(dart) Fix empty doc-comment eating next line [Jan Pilzer][]
 - enh(asciidoc) Adds support for unconstrained bold syntax (#2869) [Guillaume Grossetie][]
+- enh(c-like) Incorrect highlighting for interger suffix (#2919) [Vaibhav Chanana][]
```

Comment:
You also need to add yourself around line ~70 so it knows where to link to.

Reply:
My name is already there.

Reply:
Oh indeed. :) I forget who is first-time contributor sometimes.